modules/claims_api/app/sidekiq/claims_api/v1/poa_form_builder_job.rb

```diff
@@ -20,7 +20,7 @@ def perform(power_of_attorney_id, form_number = nil)
         power_of_attorney = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
         rep_or_org = form_number == '2122A' ? 'representative' : 'serviceOrganization'
         poa_code = power_of_attorney.form_data&.dig(rep_or_org, 'poaCode')
-        doc_type = form_number == '2122A' ? 'L075' : 'L190'
+        doc_type = form_number == '2122' ? 'L190' : 'L075'
```

Comment:
Needed to adjust v1 based on this [comment](https://jira.devops.va.gov/browse/API-37989?focusedId=3084428&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-3084428).

tl;dr - `L075` is more generic and applies to 2122 and 2122A, so if we don't have `form_number` explicitly defined, we fall back to `L075`.